pkg/didcomm/protocol/mediator/service_test.go

```diff
@@ -1432,29 +1428,6 @@ func TestGetConnections(t *testing.T) {
 		require.NoError(t, err)
 		require.Empty(t, connID)
 	})
-
-	t.Run("test get connection - db error", func(t *testing.T) {
```

Comment:
removed test?

Reply:
The method that it was testing doesn't exit anymore